catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/BoundingBoxReader.java

```diff
@@ -115,7 +123,8 @@ public String getWkt() throws CswException {
         // started
         reader.moveUp();
 
-        LOGGER.debug("Returning WKT in LON/LAT coord order: {}.", wkt);
+        wkt = convertToEPSG4326(wkt, crs);
```

Comment:
The only time we wouldn't want to convert the incoming geometry is when the CRS is EPSG:4326 and the ordering is lon / lat.  The issue here is that there are multiple CRSs for EPSG:4326 ( `urn:ogc:def:crs:OGC:1.3:CRS84`, `EPSG:4326`, `urn:x-ogc:def:crs:EPSG:6.11:4326`)